src/components/text/stories/textRules.a11y.mdx

```diff
@@ -0,0 +1,13 @@
+| **Should** be enlargable                                                                                                                                                                                                                                        | The page should be functional when only the text is magnified to 200% of its initial size. User can zoom in and out to change te size of the text. <br/> User default font size is not respected by the component.                                                                                                                                                             | Implementation: DONE<br />Tests: N/A  |
+| **Should not** exceed 80 characters in length                                                                                                                                                                                                                   |                                                                                                                                                                                                                                                                                                                                                                                | Implementation: N/A<br />Tests: N/A   |
+| **Should not** be to small                                                                                                                                                                                                                                      | Default document font size (1em) is 16px, `size` prop defaults to `medium`.                                                                                                                                                                                                                                                                                                    | Implementation: DONE<br />Tests: DONE |
+| **Should** be readable and not truncated, when user sets line height to 1.5 times font size, spacing following paragraphs to (at least) 2 times font size , letter spacing to (at least) 0.12 times font size , word spacing to (at least) 0.16 times its size. | To test, apply [the CSS](/docs/foundation-✨-typography--page#accessibility)                                                                                                                                                                                                                                                                                                   | Implementation: N/A<br />Tests: N/A   |
```

Comment:
I think this is something similar to @coderitual comment:
> I am not sure if I get it correctly. How this applies to our typography definition:
> 
> https://design.brainly.com/8adfd5f36/p/44cf68-typography
> 
> Those rules sounds more like something to be aware of when defining typography but I am not sure what actions can be made by developers when the proposed test fails. They are just using components selected by designer from the design system.

Not much can be done by SG user regarding that